packages/camel-catalog/kaoto-camel-catalog-maven-plugin/src/main/java/io/kaoto/camelcatalog/CamelCatalogProcessor.java

```diff
@@ -259,15 +267,19 @@ public String getPatternCatalog() throws Exception {
                 }
                 var catalogOp = catalogOpOptional.get();
                 if ("object".equals(catalogOp.getType()) && !catalogOp.getJavaType().startsWith("java.util.Map")
-                        && !propertySchema.has("$comment")) {
-                    propertySchema.put("$comment", "class:" + catalogOp.getJavaType());
+                        && !propertySchema.has("format")) {
+                    propertySchema.put("format", "class:" + catalogOp.getJavaType());
                 }
+
+                propertiesSchema.put(propertyName, propertySchema);
```

Comment:
@igarashitm This line says the `put(String, JsonNode)` method is deprecated, what can I use instead?

![image](https://github.com/KaotoIO/kaoto-next/assets/16512618/853e6cb0-7ac7-49fb-90ea-cf8a313591ac)

Reply:
Yes, thanks 💪